File: app/src/main/java/vn/loitp/app/activity/customviews/layout/constraintlayout/fabfollowswiidget/FabFollowWidgetActivity.java
package vn.loitp.app.activity.customviews.layout.constraintlayout.fabfollowswiidget;

import android.os.Bundle;

import com.core.base.BaseFontActivity;

import loitp.basemaster.R;

public class FabFollowWidgetActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_fab_follow_widget;
    }

}

File: app/src/main/java/vn/loitp/app/activity/customviews/layout/constraintlayout/fabfollowswiidget/FabFollowWidgetActivity.kt
package vn.loitp.app.activity.customviews.layout.constraintlayout.fabfollowswiidget

import com.annotation.IsFullScreen
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import vn.loitp.app.R

@LogTag("FabFollowWidgetActivity")
@IsFullScreen(false)
class FabFollowWidgetActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_fab_follow_widget
    }
}
